wds-wp-json-api-connect.php

```diff
@@ -527,7 +529,15 @@ public function cache_api_description_for_json_url() {
 				return $this->json_desc;
 			}
 
-			$this->response = wp_remote_get( $this->args['json_url'] );
+			$get_args = array(
+				'headers' => array(),
+			);
+
+			if ( strlen( $this->require_login_key ) && strlen( $this->require_login_token ) ) {
+				$get_args['headers'][ $this->require_login_key ] = $this->require_login_token;
```

Comment:
> Line 537 is using object properties when the 'require_login_key' and 'require_login_token' are set as values of the arms property

Ah I see - they are also set as accessible via the magic __get on 897 and 898. Blinded me to the use of $this->args right above. Will change that.